### PR TITLE
Update lerna to use yarn

### PR DESCRIPTION
**Pull request by hzoo**
Updates to use yarn in boostrap (doesn't save too much for us since we don't have that many external deps: 8.9s vs 11.4s)

---

Bootstrapping 111 packages
Encountered a cycle in the dependency graph. This may cause instability!
Encountered a cycle in the dependency graph. This may cause instability!
Encountered a cycle in the dependency graph. This may cause instability!

### CODE REVIEW

File: packages/babel-core/package.json [hunk 1]
```diff
@@ -21,10 +21,6 @@
     "babel-core",
     "compiler"
   ],
-  "scripts": {
```
Comment (hzoo) [inline packages/babel-core/package.json]: unnecessary (from when babel-core was babel)

**Comment by hzoo** [comment]
Hmm ok this setup worked https://travis-ci.org/lerna/lerna-test/builds/207012406

**Comment by aaronang** [comment]
I reproduced this error locally. Here is the readable version:

```bash
$ make bootstrap          
                                                                                                                                                
update-lerna
make clean-all
rm -rf node_modules
rm -rf packages/*/node_modules
make clean
rm -rf packages/*/test/tmp
rm -rf packages/*/test-fixtures.json
rm -rf packages/babel-polyfill/browser*
rm -rf packages/babel-polyfill/dist
rm -rf coverage
rm -rf packages/*/npm-debug*
yarn
yarn install v0.21.3
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
[4/4] 📃  Building fresh packages...
✨  Done in 103.51s.
./node_modules/.bin/lerna bootstrap --npm-client=yarn
Lerna v2.0.0-beta.38
Bootstrapping 111 packages
Encountered a cycle in the dependency graph. This may cause instability!
Encountered a cycle in the dependency graph. This may cause instability!
Encountered a cycle in the dependency graph. This may cause instability!
Preinstalling packages
Installing external dependencies
NpmUtilities.installInDir     ("./packages/babel-plugin-transform-es2015-unicode-regex", ["regexpu-core@^4.0.2"], {"client":"yarn"})
Command exited with status 1: yarn install
Errored while running BootstrapCommand.execute
Command exited with status 1: yarn install
Waiting for 3 child processes to exit. CTRL-C to exit immediately.
make: *** [bootstrap] Error 1
```

**Comment by hzoo** [comment]
curious @mathiasbynens does `unicode-property-value-aliases require Node >=6.` or should it be 4?

**Comment by mathiasbynens** [comment]
@hzoo I just published v1.2.2 with official™ Node.js v4+ support.

**Comment by hzoo** [comment]
Maybe just PR lerna to just use `--no-lockfile`? Going to close and make a new PR for npmignore normalization

**Pull request closed without merging**